### PR TITLE
Fix exclusiveMinimum according to latest JSON Schema spec. Remove custom format. 

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

**Comment by jmcanterafonseca** [comment]
@aarranz please also review

File: specs/StreetLighting/StreetlightGroup/schema.json [hunk 1]
```diff
@@ -48,14 +48,14 @@
...
-                  {"format": "ext-date-yearless"}
+                  {"$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/DateYearLess"}
                 ]
               },
               "to": {
                 "type": "string",
                 "oneOf": [
                   {"format": "date"},
-                  {"format": "ext-date-yearless"}
+                  {"$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/DateYearLess"}
```
Comment (juliozinga) [inline specs/StreetLighting/StreetlightGroup/schema.json]: This definition really exists? 'DateYearLess'

Comment (jmcanterafonseca) [reply specs/StreetLighting/StreetlightGroup/schema.json]: now, it was missed

**Comment by jmcanterafonseca** [comment]
landing as no response got for days.

**Comment by aarranz** [comment]
Sorry for not answering on time, but yes, It's ok for me.

**Pull request merged**